### PR TITLE
Part4

### PR DESCRIPTION
**Pull request by KottV**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -203,26 +203,23 @@ DAC::DAC()
...
     int outCurrent;
-    int maxOutTemp;
-    PowerUnit();
+    int maxOutTemp { 80 };
+    PowerUnit() : weight(1.2f), inVolt(220), outCurrent(1){}
+    void printStatus()
+    {
+        std::cout<<"Power Unit Status:"<<std::endl;
+        std::cout<<"inV:"<<inVolt<<std::endl;
+        std::cout<<"outV:"<<outVolt<<std::endl;
```
Comment (matkatmusic) [inline main.cpp]: Improve the readability by adding spaces around `<<`
- [ ] revise

File: main.cpp [hunk 2]
```diff
@@ -592,15 +589,8 @@ bool MonitorController::toggleCrossfeed(bool status)
...
-    CanPlant::OfficeWorker worker;
-    
-    Doc.examineAnimal(2, 0.5f, Pusya);
-    Pusya.mew(3);
-    
-    std::cout << "What is doc's name? - "<<Doc.name<<"\n";
-    std::cout << "How much boxes reserved? " << worker.reserveGood(100, 1) << std::endl;
+    PowerUnit powerunit;
+    powerunit.printStatus();
+
```
Comment (matkatmusic) [inline main.cpp]: From project3.3:
```
 3) Instantiate 1 or 2 instances of each of your user-defined types in the main() function

 4) call some of your UDT's amended member functions in main().
 
 5) add some std::cout statements in main() that print out your UDT's member 
variable values or values returned from your UDT member functions (if they return values)
 ```
Your `main()` is lacking a lot of code.  
Instantiate your objects and call their member functions to make sure your code works.
- [ ] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]
You only changed one class to conform with this instruction:
```
1) initialize some of your member variables either in-class or in the Constructor 
member initializer list.

 2) make some of your member functions use those initialized member variables 
via std::cout statements.

```
Please revise all of your classes, including the nested ones, to demonstrate to me that you understand these concepts.

**Comment by KottV** [comment]
I pushed the updated code. It has more outputs of UDT members and all constructors are in-class (except "Cat").
Also I played with std::cout as you can see. That's way I'd like to know what is good and what is not so.

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**